conf/config.json

```diff
@@ -92,17 +92,16 @@
         },
         "lifecycle": {
             "auth": {
-                "type": "vault",
-                "vault": {
-                    "roleName": "lifecycle-role",
+                "type": "assumeRole",
+                "role": "lifecycle-role",
+                "sts": {
                     "host": "127.0.0.1",
                     "port": 8800,
-                    "serviceCredentials": {
-                        "type": "account",
-                        "account": "lifecycle"
-                    },
-                    "adminPort": 8600,
-                    "adminCredentialsFile": "./admincredentials.json"
+                    "serviceUser": "lifecycle"
```

Comment:
This seems to be taking advantage of `authdata.json`, which is made to hold accounts. I believe `authdata.json` was not meant to store credentials but rather be a "central" repository of accounts where Vault is not available (so typically Zenko or standalone open source cloudserver).

Its use here is a stretch because we're introducing a made up canonical id. Just adding an access key and secret key fields here would avoid subverting `authdata.json`'s initial purpose (which is less and less relevant now that Artesca uses Vault).

Reply:
`accessKey` and `secretKey` in the `config.json` can work and avoid using the `authdata.json` for purposes it wasn't intended for